### ISSUE TITLE
Upgrade Lerna to 7.x

### ISSUE THREAD

**Issue by EmiM** [TryQuiet/quiet#2029]

**Comment by EmiM** [comment]
Kacper started working on this some time ago: https://github.com/TryQuiet/quiet/pull/1930

**Comment by holmesworcester** [comment]
Someone also suggested npm workspaces as an alternative, I think. Just noting this here.

**Comment by EmiM** [comment]
Yes, Wiktor suggested that and we considered it but lerna has some additional features that we may need.
https://stackoverflow.com/questions/64909635/is-lerna-needed-anymore-with-npm-workspaces

Upgrading lerna requires some work but also giving up using lerna requires work.
